src/views/product-integrations-landing/index.tsx

```diff
@@ -4,26 +4,31 @@ import { IntegrationsSearchProvider } from './contexts/integrations-search-conte
 import { type Integration } from 'lib/integrations-api-client/integration'
 
 import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import { TryHcpCalloutSidecarPlacement } from 'components/try-hcp-callout/components'
 import { type SidebarProps } from 'components/sidebar'
 import { type BreadcrumbLink } from 'components/breadcrumb-bar'
 
-interface ViewProps {
+export interface ViewProps {
 	integrations: Array<Integration>
 	sidebarNavDataLevels: Array<SidebarProps>
 	breadcrumbLinks: Array<BreadcrumbLink>
+	productSlug: string
 }
 
 export default function ProductIntegrationsLanding({
 	integrations,
 	sidebarNavDataLevels,
 	breadcrumbLinks,
+	productSlug,
 }: ViewProps) {
 	return (
 		<IntegrationsSearchProvider integrations={integrations}>
 			<SidebarSidecarLayout
 				sidebarNavDataLevels={sidebarNavDataLevels}
 				breadcrumbLinks={breadcrumbLinks}
-				sidecarSlot={<></>}
+				sidecarSlot={
+					<TryHcpCalloutSidecarPlacement productSlug={productSlug} />
```

Comment:
I don't think we want this on the landing page, actually!

It looks like we actually shouldn't even have a sidecar slot for anything on that landing page now that I am re-reviewing the designs, but don't worry about that for now (I'll make another ticket and chat with design to figure out what they're looking for there).

Reply:
Ah great catch - i've removed the callout from `/<product>/integrations` landing pages ✅ 🔪  will keep an eye on the board for that new ticket ☺️